lib/components/StatusTrend/StatusTrend.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { FC } from "react";
+
+import {
+  LabelWrapper,
+  StatusTrendWrapper,
+  StyledPrimaryLabel,
+  StyledSecondaryLabel,
+} from "./StatusTrend.styled";
+import { StatusTrendProps } from "./StatusTrend.types";
+import { StatusTrendSymbol } from "./StatusTrendSymbol";
+
+const StatusTrend: FC<StatusTrendProps> = ({
+  labelPrimary,
+  labelSecondary,
+  direction = "neutral",
```

Comment:
we should get the value as a number, and calculate the direction based on it

Reply:
This one I must disagree. I separated this because, in our case, a "negative" number can be a good thing (as in "accidents", then it's shown as green), but can be a bad thing (as in "number of sales"), so the number/sign shouldn't be coupled with the icon/color. I hope I was able to explain ;-)

Reply:
Also, I think it's better to have the label as children (and I've tried to follow the same logic as used on the StatusTag, etc.), it's more versatile, because we can use labels as "-10" or "-10%" or "-10 pts", etc.